data/json/professions.json

```diff
@@ -6322,5 +6344,57 @@
       "male": [ "briefs" ],
       "female": [ "panties", "bra" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "rv_hunter",
+    "name": "RV Hunter",
+    "description": "Ever since you were a child you loved hunting and traveling; so when you scraped enough money, you bought yourself an old RV, a gun, and hit the road.  You've been all over the country, hunting for good game.  Unfortunately the country is all over this time, and even more unfortunately you've lost your ol' reliable in the chaos.  Now you gotta find a new reliable.",
+    "points": 4,
+    "proficiencies": [ "prof_driver", "prof_basic_engines" ],
+    "skills": [
+      { "level": 3, "name": "rifle" },
+      { "level": 3, "name": "gun" },
+      { "level": 2, "name": "swimming" },
+      { "level": 4, "name": "driving" },
+      { "level": 4, "name": "mechanics" }
+    ],
+    "vehicle": "rv",
+    "items": {
+      "both": {
+        "items": [ "knit_scarf", "socks", "boots_steel", "pants_cargo", "jacket_flannel", "hat_hunting", "wristwatch", "back_holster" ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "sheath", "contents-item": "knife_hunting" },
+          { "item": "tank_top", "variant": "tank_top_camo" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    },
+    "missions": [ "MISSION_HUNTER", "MISSION_FIND_RIFLE" ]
+  },
+  {
+    "type": "profession",
+    "id": "heist_driver",
+    "name": "Heist Driver",
+    "description": "For as long as you can remember, you have lived a double life: a normal driver by the day and a heist driver in the night.  Now there are no cops and no law, but even after the Cataclysm, you drive.",
+    "points": 5,
+    "proficiencies": [ "prof_driver", "prof_basic_engines", "prof_athlete_basic" ],
+    "skills": [
+      { "level": 6, "name": "driving" },
+      { "level": 4, "name": "mechanics" },
+      { "level": 3, "name": "unarmed" },
```

Comment:
Why is the heist driver also a hand to hand combat expert?

Reply:
I think realistically he would be able to handle himself in hand to hand combat. Wouldn't be plausible that he engaged in melee but never unarmed

Reply:
It would make more sense to leave that for people to select in hobbies if they wanted him to have combat experience.  Basically the way I look at professions is that if something isn't key to the theme I'm creating then I'm limiting player choice by adding more skill levels.

Reply:
alright when you put it like that. Lowering melee to 2 so he still can use the hammer but removing the unarmed, maybe people should take street fighting background for that. lowering the selecting point by 1 also

Reply:
keeping the melee since he is a criminal after all